.github/workflows/ci.yml

```diff
@@ -1,6 +1,11 @@
 on:
   workflow_call:
     inputs:
+      nightly:
+        description: When set to true, the workflow will run with nightly reports
```

Comment:
Maybe just change the description to ` When set to true, the workflow will run the nightly setup` other than that looks good